downstream/modules/platform/proc-gw-create-roles.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-gw-create-roles_{context}"]
+
+= Creating a role
+
+ifdef::auto-exec
```

Comment:
Have you tested this Donna? I always thought you needed a [] at the end of the ifdef/endif statement.